### PR TITLE
Nerfed praying mantises

### PR DESCRIPTION
**Pull request by sonphantrung**
#### Summary
Balance "Nerfed Praying Mantises"

#### Purpose of change
Based on my experience, Praying mantises are far more dangerous than giant ladybugs, namely because of their extreme stats. While most of my early game powerful characters are able to take down ladybugs with a small scratch, mantises bleed my dudes like CRAZY!!!!

#### Describe the solution
- Praying mantis: Halved its melee skills
- Mega mantis: Halved its HP (2400 seems way OVERTUNED, compared to the titan stag beetle, which has 1400 HP)

#### Describe alternatives you've considered
NONE

#### Testing
If it doesn't throw errors, it should be good

#### Additional Context
Contributes to #55409

### CODE REVIEW

**Comment by sonphantrung** [comment]
What the heck, why do tests throw errors at me?

**Comment by anoobindisguise** [comment]
Basic praying mantises actually do *more* damage than their evolved counterparts. This is because of the entry 
    "melee_damage": [ { "damage_type": "cut", "amount": 8 } ],
The order of operation is that it copies all the data from the evolved mantis, then it *adds* 8 more cut damage rather than replacing its evolved form's cut damage. I'd try changing this to a suitable negative number and see if that fixes it as it is unambiguously a bug (heh)

**Comment by LyleSY** [comment]
That's also consistent with how I understand the code to work. @anoobindisguise can you confirm?

File: data/json/monsters/insect_spider.json [hunk 1]
```diff
@@ -2569,7 +2569,7 @@
     "name": { "str": "great mantis", "str_pl": "great mantises" },
     "description": "This ferocious cow-sized mantis stalks the fields of New England.  It's probably capable of killing a moose with its powerful arms.",
     "copy-from": "mon_mantis_giant",
-    "proportional": { "hp": 20, "speed": 0.25, "vision_day": 2 },
+    "proportional": { "hp": 10, "speed": 0.25, "vision_day": 2 },
```
Comment (LyleSY) [inline data/json/monsters/insect_spider.json:2572]:
```suggestion
    "proportional": { "hp": 3, "speed": 0.25, "vision_day": 2 },
```
Per the balance doc, this is a better fit for something using blob magic but not an endgame supernatural boss https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/GAME_BALANCE.md

**Review by LyleSY — CHANGES_REQUESTED** [review]
Definitely an improvement, and I can live with the lower skill, but ideally the HP should be within the range in the balance doc

**Review by LyleSY — APPROVED** [review]
Solid improvement, good work

**Pull request merged**